src/cdk/overlay/overlay-ref.ts

```diff
@@ -309,6 +309,20 @@ export class OverlayRef implements PortalOutlet, OverlayReference {
     this._updateElementDirection();
   }
 
+  /** Add a CSS class or an array of classes to the overlay pane. */
+  addPanelClass(classes: string | string[]) {
```

Comment:
Could you explicitly add the `void` return types? Our docs generation isn't good with the implicit types